ts/Core/Axis/Axis.ts

```diff
@@ -7250,10 +7250,13 @@ class Axis implements AxisComposition, AxisLike {
         const isLinked = axis.isLinked;
         const ticks = axis.ticks;
         const slideInTicks = axis.chart.hasRendered && isNumber(axis.oldMin);
+        const tickInterval = axis.tickInterval;
 
         // Linked axes need an extra check to find out if
         if (!isLinked ||
-            (pos >= (axis.min as any) && pos <= (axis.max as any))
+            (pos >= (axis.min as any) && pos <= (axis.max as any)) ||
+            axis.grid &&
+            (pos >= (axis.min as any) - 0.5 * tickInterval && pos <= (axis.max as any) + 0.5 * tickInterval)
```

Comment:
Hmm.. where this `0.5` comes from?

Reply:
To avoid collisions with the column title text we have to hide the grid name exactly at the middle of the first line.

Reply:
I'm sorry, I don't quite understand.. Anyway, should be moved out from here if possible :)